hardware/raspberrypi/booteeprom.md

```diff
@@ -44,23 +49,24 @@ sudo systemctl unmask rpi-eeprom-update
 
 The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
 
-## Write protection of EEPROM
-
-There is no software write protection for the boot EEPROM but there will be a mechanism in Raspberry Pi OS to skip any future updates to the EEPROM. However, it is possible to physically write-protect both EEPROMs via a simple resistor change on the board. Details will be published in the [schematics](./schematics/README.md).
-
-EEPROM image files contain a small user-modifiable config file, which may be modified using the `rpi-eeprom-config` script included in the `rpi-eeprom` package. See the [Bootloader Configuration Page](./bcm2711_bootloader_config.md) for configuration details.
+Note that by default, updating the bootloader (automatically or manually) will retain any custom configuration options of the previous installed version. You can override the migration by manually updating and using the `-d` option with `rpi-eeprom`. This will force the updated bootloader to use its inbuilt defaults.
```

Comment:
Still need to change `rpi-eeprom` -> `rpi-eeprom-update` here?

Reply:
I must have changed the wrong one...